go.mod

```diff
@@ -1,3 +1,7 @@
 module github.com/googleapis/enterprise-certificate-proxy
 
 go 1.18
+
+replace github.com/googleapis/enterprise-certificate-proxy/utils => ./utils
```

Comment:
I don't think you need this. You just need to make sure the util package is under your local directory: $GOPATH/src/github.com/googleapis/enterprise-certificate-proxy/utils

Reply:
Go mod has somewhat replaced GOPATH (and no assumptions should be made on GOPATH as that is very user and env specific). We should not use replace syntax either however.